pkg/controllers/nodeclaim/disruption/suite_test.go

```diff
@@ -116,7 +116,7 @@ var _ = Describe("Disruption", func() {
 	})
 	It("should remove multiple disruption conditions simultaneously", func() {
```

Comment:
I think we should testing that we can trigger drift by updating the expireAfter field